contents/docs/advanced/proxy.mdx

```diff
@@ -1,109 +1,168 @@
--   [Next.js middleware](/docs/advanced/proxy/nextjs-middleware)
--   [nginx](/docs/advanced/proxy/nginx)
--   [node](/docs/advanced/proxy/node)
--   [Railway](/docs/advanced/proxy/railway)
--   [Remix](/docs/advanced/proxy/remix)
--   [SvelteKit](/docs/advanced/proxy/sveltekit)
--   [Vercel](/docs/advanced/proxy/vercel)
--   [Nuxt](/docs/advanced/proxy/nuxt)
--   [Pomerium](/docs/advanced/proxy/pomerium)
+### Managed reverse proxy (recommended)
 
-## Best practices
+Point a subdomain to PostHog and we handle SSL certificates, routing, and maintenance automatically. If you run into issues, we provide support.
+
+This option requires our [platforms add-ons](/platform-packages) and is the fastest way to get a proxy running.
+
+<div>
+
+<CallToAction to="/docs/advanced/proxy/managed-reverse-proxy">Set up managed reverse proxy</CallToAction>
+
+</div>
+
+### Deploy your own proxy
+
+If you need full control or have existing infrastructure you want to use, you can deploy your own reverse proxy. If you use a self-hosted proxy, PostHog can't help troubleshoot configuration issues.
+
+Choose your platform and follow the setup guide:
+
+<ProxyPlatforms />
+
+### Need help choosing?
 
--   We require that the proxy sets the `Host` header to the same host it's calling. Check the guides above on how to do that for several proxies.
--   Don't use a subdomain that includes `posthog`, `analytics`, `tracking`, or other similar words which might cause events to be blocked.
--   Make sure to [pass the proper `ui_host` parameter](/docs/libraries/js/config) when initializing PostHog, so that links to PostHog point to the correct host (like `us.posthog.com`). This is required for tasks like authenticating the toolbar.
--   <ProxyPathNamesWarning />
+<ProxyDecisionTree />
 
-## Reverse proxy requirements
+## Configuration overview
 
-If you want to use an alternative reverse proxy that we have not documented, it must meet the following requirements:
+After your proxy is running, you'll need to configure a few things. Each setup guide covers these steps in detail, but here's what to expect:
+
+1. **Choose your subdomain carefully**. Ad blockers look for obvious terms. Avoid words like `posthog`, `analytics`, `tracking`, or `telemetry` in your subdomain or paths. Use something neutral like `e.yourdomain.com` or `ph.yourdomain.com`.
+2. **Update your PostHog SDK configuration**. Point your SDK to your proxy domain instead of PostHog's domain. Always set both `api_host` (your proxy) and `ui_host` (PostHog's actual domain) so features like the toolbar work correctly:
+
+  ```js
+  posthog.init('<ph_project_api_key>', {
+    api_host: 'https://e.yourdomain.com',
+    ui_host: 'https://us.posthog.com' // or https://eu.posthog.com
+  })
+  ```
+
+3. **Verify events are flowing**. Open your browser's network tab, trigger an event, and confirm requests to your proxy return `200 OK`. Then check your PostHog dashboard to verify events appear.
+
+## Technical requirements
```

Comment:
Should the information below really stay on this page, or should they go to `/docs/advanced/proxy/managed-reverse-proxy`?

I think these are a little heavy for the initial install and configure.